### PR TITLE
Add leetcode 169

### PR DESCRIPTION
**Pull request by zmzhouXJTU**

### CODE REVIEW

File: src/169._Majority_Element.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            if dic[num] > len(nums)//2:
+                return num
+            else:
+                dic[num] += 1 
+```
+
+#### Approach 2: Sorting
+
+* Time complexity : O(nlogn)
+* Space complexity : O(1)
```
[the remark this answers is not in the document]
Comment (zmzhouXJTU) [reply src/169._Majority_Element.md]: Oh, yes. I will correct it later

File: src/169._Majority_Element.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        """
+        Given an array of size n, find the majority element. The majority element is the element that appears more than ⌊ n/2 ⌋ times.
+
+        Args:
+            nums(List[int]): the name of array of integers
+
+        Returns: 
+            the element that appears more than ⌊ n/2 ⌋ times(int)
+        """
+        return sorted(nums)[len(nums)//2]
```
Comment (lingmagirl) [inline src/169._Majority_Element.md]: `len(nums)//2` -> `len(nums) // 2`

File: src/169._Majority_Element.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            nums(List[int]): the name of array of integers
+
+        Returns: 
+            the element that appears more than ⌊ n/2 ⌋ times(int)
+        """
+        dic = {}
+        for num in nums:
+            if num not in dic:
+                dic[num] = 1
+            if dic[num] > len(nums)//2:
```
Comment (lingmagirl) [inline src/169._Majority_Element.md]: `len(nums)//2` -> `len(nums) // 2:`

Comment (zmzhouXJTU) [reply src/169._Majority_Element.md]: OK，I have corrected it just now. Much thanks for your careful review

**Pull request merged**